scout/server/blueprints/dashboard/controllers.py

```diff
@@ -191,13 +227,20 @@ def get_general_case_info(adapter, institute_id=None, slice_query=None):
     for total_cases, case in enumerate(cases, 1):
         case_ids.add(case["_id"])
         if case.get("phenotype_terms"):
-            phenotype_cases += 1
+            case_counter["phenotype"] += 1
         if case.get("causatives"):
-            causative_cases += 1
+            case_counter["causative"] += 1
         if case.get("suspects"):
-            pinned_cases += 1
+            case_counter["pinned"] += 1
         if case.get("cohorts"):
-            cohort_cases += 1
+            case_counter["cohort"] += 1
+        if case.get("tags"):
+            case_counter["tagged"] += 1
+            stat_tags = ["provisional", "diagnostic", "incidental", "carrier"]
```

Comment:
Why collecting these tags only? There are more tag types available for cases.. And perhaps other tags could be added in the future. Why not unifying so that these tags are collected from [here]?(https://github.com/Clinical-Genomics/scout/blob/ad8b1342ad53c93b01f3a2dbd15c9c3caa5200c9/scout/constants/case_tags.py#L63)